static/app/views/integrationOrganizationLink.tsx

```diff
@@ -204,46 +204,6 @@ export default class IntegrationOrganizationLink extends AsyncView<Props, State>
     );
   }
 
-  customOption = orgProps => {
-    const organization = this.getOrgBySlug(orgProps.value);
-    if (!organization) {
-      return null;
-    }
-    return (
-      <components.Option {...orgProps}>
-        <IdBadge
-          organization={organization}
-          avatarSize={20}
-          displayName={organization.name}
-          avatarProps={{consistentWidth: true}}
-        />
-      </components.Option>
-    );
-  };
-
-  customValueContainer = containerProps => {
-    const valueList = containerProps.getValue();
-    // if no value set, we want to return the default component that is rendered
-    if (valueList.length === 0) {
-      return <components.ValueContainer {...containerProps} />;
-    }
-    const orgSlug = valueList[0].value;
-    const organization = this.getOrgBySlug(orgSlug);
-    if (!organization) {
-      return <components.ValueContainer {...containerProps} />;
-    }
-    return (
-      <components.ValueContainer {...containerProps}>
-        <IdBadge
-          organization={organization}
-          avatarSize={20}
-          displayName={organization.name}
-          avatarProps={{consistentWidth: true}}
-        />
-      </components.ValueContainer>
-    );
-  };
-
```

Comment:
These custom sub-components are no longer necessary — `SelectControl` accepts non-string elements for option labels.